storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3451,6 +3454,8 @@ static void rocksdb_update_table_stats(
 
     io_perf_read.bytes = table_handler->m_io_perf_read.bytes.load();
     io_perf_read.requests = table_handler->m_io_perf_read.requests.load();
+    lock_wait_timeout_stats = table_handler->m_lock_wait_timeout_counter.load();
+    table_handler->m_lock_wait_timeout_counter.clear();
```

Comment:
good point. Looks like you may use https://en.wikipedia.org/wiki/Compare-and-swap perhaps?